packages/nc-gui/assets/style.scss

```diff
@@ -591,7 +604,7 @@ a {
 }
 
 .ant-popover-inner {
-  padding: 0 !important;
+  @apply rouded-lg !p-0;
```

Comment:
typo - `rounded-lg`

Reply:
**FIXED in Below Commit**

[fix : fixed typo](https://github.com/nocodb/nocodb/pull/7181/commits/40025173e0006f2acefa2a9bd0d10647fd554b25)
 
[4002517](https://github.com/nocodb/nocodb/pull/7181/commits/40025173e0006f2acefa2a9bd0d10647fd554b25)